src/node_process_methods.cc

```diff
@@ -163,6 +163,10 @@ static void Cwd(const FunctionCallbackInfo<Value>& args) {
   size_t cwd_len = sizeof(buf);
   int err = uv_cwd(buf, &cwd_len);
   if (err) {
+    if (err == UV_ENOENT) {
```

Comment:
The error that's thrown already has an "ENOENT" error code. This shouldn't be necessary.

Reply:
It _does_, but from the linked issue it is often non-obvious where the error actually came from. By applying this change it makes it clearer _why_ this `ENOENT` error happened.